engine/gui/src/dmsdk/gui/gui.h

```diff
@@ -57,6 +57,14 @@ namespace dmGui
      */
     typedef struct Script* HScript;
 
+    /*#
+     * A handle to a texture source, which can be a pointer to a resource,
+     * a dmGraphics::HTexture or a dynamic texture created from a gui script.
+     * @name HTextureSource
+     * @type typedef
+     */
+    typedef uint64_t HTextureSource;
```

Comment:
Another idea would be to represent this as a struct that also contains the 'NodeTextureType' 🤔

Reply:
Sounds like a good idea (for a later PR)